packages/app-info/README.md

```diff
@@ -1,18 +1,17 @@
-
 ## @dotcom-reliability-kit/app-info
 
 A utility to get application information (e.g. the system code) in a consistent way. This module is part of [FT.com Reliability Kit](https://github.com/Financial-Times/dotcom-reliability-kit#readme).
 
-  * [Usage](#usage)
-    * [`appInfo.commitHash`](#appinfocommithash)
-    * [`appInfo.environment`](#appinfoenvironment)
-    * [`appInfo.region`](#appinforegion)
-    * [`appInfo.releaseDate`](#appinforeleasedate)
-    * [`appInfo.releaseVersion`](#appinforeleaseversion)
-    * [`appInfo.systemCode`](#appinfosystemcode)
-  * [Contributing](#contributing)
-  * [License](#license)
-
+- [Usage](#usage)
```

Comment:
**suggestion:** there are a few Markdown styling changes in here which make the README diverge from the rest of the packages (e.g. spacing and changing `*` to `-`). I think there's some work for us to do to enforce the consistency we currently have but in the meantime could you revert these changes?

Reply:
gah, sorry, VS Code must have formatted it without me noticing